index.html

```diff
@@ -1,256 +1,292 @@
-
-    <script src="node_modules/jquery/dist/jquery.min.js"></script>
-    <script src="node_modules/bootstrap/dist/js/bootstrap.min.js"></script>
-
-    <script src="node_modules/webrtc-adapter/out/adapter.js"></script>
-    <script src="node_modules/getusermedia-js/dist/getUserMedia.min.js"></script>
-    <script src="node_modules/socket.io-client/dist/socket.io.js"></script>
-
-    <script src="dist/infragram.js"></script>
-
-    <script type="x-shader/x-vertex" id="shader-vs"></script>
-    <script type="x-shader/x-fragment" id="shader-fs-template"></script>
-    <script type="x-shader/x-fragment" id="shader-fs"></script>
-
-  </head>
-
-  <body>
-
-    <div class="container page" style="padding-top:10px;">
-
-       <div class="modal" tabindex="-1" role="dialog" id="preset-modal">
-         <div class="modal-dialog modal-lg" role="document">
-           <div class="modal-content">
-             <!--<div class="modal-header"></div>-->
-             <div class="modal-body">
-               <h3>Quick start</h3>
-               <p>Choose from these presets to get up and running quickly. These auto-load common 2. Analysis and 3. Colorize settings so you don't have to manually choose them.</p>
-                 <table class="table">
-                   <tr>
-                     <td width="30%"><b>Raw</b></td>
-                     <td width="30%">Just what your camera sees.</td>
-                     <td><a class="btn btn-primary" id="preset_raw">Begin</a></td>
-                   </tr>
-                   <tr>
-                     <td><b>NDVI for <span style="color:blue;">BLUE</span> filters</b></td>
-                     <td>basic plant analysis for <a href="http://store.publiclab.org/product/infragram-diy-filter-pack">Infragram filter packs</a> and <a href="http://store.publiclab.org/product/infragram-webcam">webcams</a></td>
-                     <td><a class="btn btn-primary" id="preset_ndvi_blue">Basic</a>
-                         <a class="btn btn-success" id="preset_ndvi_blue_color">Colorized</a></td>
-                   </tr>
-                   <tr>
-                     <td><b>NDVI for <span style="color:red;">RED</span> filters</b></td>
-                     <td>basic plant analysis for <a href="http://store.publiclab.org/products/infragram-point-shoot-plant-cam">Infragram Point &amp; Shoot cameras</a></td>
-                     <td><a class="btn btn-primary" id="preset_ndvi_red">Basic</a>
-                         <a class="btn btn-success" id="preset_ndvi_red_color">Colorized</a></td>
-                   </tr>
-                 </table>
-                 <p>Post a research note at <a href="http://publiclab.org">PublicLab.org</a> and email <a href="mailto:plots-dev@googlegroups.com">plots-dev@googlegroups.com</a> to suggest new presets.</p>
-
-	    </div>
-            <div class="modal-footer">
-              <a data-dismiss="modal" data-toggle="modal" class="btn">cancel</a>
-            </div>
```

Comment:
@shubhangi013 I don't think you've disabled your linter/formatter.

Reply:
> @shubhangi013 I don't think you've disabled your linter/formatter.


I think it's not because of the linter...because that I've disabled but, because of indentation difference in my IDE. and here are the after and before results :
Before:
![Screenshot from 2022-03-03 21-39-38](https://user-images.githubusercontent.com/78342516/156604313-daa02c1b-be60-4ed5-981d-b4990b955aaa.png)
![Screenshot from 2022-03-03 21-39-24](https://user-images.githubusercontent.com/78342516/156604325-1c6b1bee-2619-44e0-ae1d-5bca62cebdc1.png)


After:
![Screenshot from 2022-03-03 21-34-51](https://user-images.githubusercontent.com/78342516/156604031-392cfa18-3ee3-4b7a-886e-a21975d02eaa.png)
![Screenshot from 2022-03-03 21-34-01](https://user-images.githubusercontent.com/78342516/156604040-c6e19f60-32f0-4c16-81ed-b822a5fa45cd.png)

Reply:
@shubhangi013 Did you undo the changes your linter added before?

Reply:
Usually the linter/formatter is the problem in this case but since you've disabled it, I'm not sure what the issue is.

Reply:
See if the issue helps https://github.com/github/VisualStudio/issues/1524#issuecomment-503909421

Reply:
> See if the issue helps https://github.com/github/VisualStudio/issues/1524#issuecomment-503909421

Yes! Thank you. I'll look at it. It seems to be an indentation problem 😅